packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -353,7 +353,8 @@ type MessagesType = {
   'copy.bank_linked_error_checkingtitle': 'Please link a Checking Account.'
   'copy.bank_linked_error_checking': 'Your bank may charge you extra fees if you buy cyrpto without a checking account.'
   'copy.bank_linked': 'Your bank account is now linked to your Blockchain.com Wallet'
-  'copy.bank_linked_title': 'Bank Linked'
+  'copy.bank_linked.name': 'Your bank {bankName} account is now linked to your Blockchain.com Wallet'
```

Comment:
i don't think it makes sense to have `bank` in here.

`Your bank Chase account is now linked....` vs `Your Chase account is now linked....`

Reply:
@schnogz yeah make sense, done ✅ please do re-review.